src/wallet/ComethWallet.ts

```diff
@@ -124,8 +124,29 @@ export class ComethWallet {
     return await this.sendTransaction(tx)
   }
 
+  public async removeOwner(owner: string): Promise<SendTransactionResponse> {
+    const tx = await safeService.prepareRemoveOwnerTx(
+      this.getAddress(),
+      owner,
+      this.provider
+    )
+
+    const localStorageSigner = window.localStorage.getItem(
+      `cometh-connect-${this.getAddress()}`
+    )
+
+    if (
+      localStorageSigner &&
+      JSON.parse(localStorageSigner).signerAddress === owner
+    )
+      window.localStorage.removeItem(`cometh-connect-${this.getAddress()}`)
```

Comment:
What happen if we don't do that ?

If the tx failed and we removed the key user what are the consequences ?